### DISCUSSION TITLE
How to load the model and continue for training?

### DISCUSSION THREAD

**Discussion by Xrondev** [Nixtla/neuralforecast#1064] [Q&A] [answered]
I am quite new to time-forecasting tasks and those algorithms, please help me.
I am using the Transformer-based algorithms like the TFT and the PatchTST, I checked the doc about the "save and load model" related part but I think it only done a predict action
```
nf2 = NeuralForecast.load(path='./checkpoints/test_run/')
Y_hat_df = nf2.predict().reset_index()
Y_hat_df.head() 
```
I tried to load it and perform cross_validation() again but it seems the model started to training from begging;
   

I used the `nf.save()` to save my model, but I do not know how to "continue the training" using that model, for example, I trained the model with hourily data from 2020-10-21 to 2024-5-31, and now I want to extend the time to 2024-6-31, can I use the saved model for "continue training"?
if yes, can you please give me some simple code demo (or psuedo-code)? Thanks!

**Comment by jmoralez** [comment, marked as answer]
Hey. If you run fit/cross_validation again it continues training the same models unless you set `use_init_models=True`.

**Reply by Xrondev** [reply]
Thanks.
